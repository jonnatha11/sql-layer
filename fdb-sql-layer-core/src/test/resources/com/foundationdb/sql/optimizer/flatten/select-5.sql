SELECT name FROM child WHERE pid IN (SELECT parent.id FROM parent,child WHERE parent.name = 'foo')
