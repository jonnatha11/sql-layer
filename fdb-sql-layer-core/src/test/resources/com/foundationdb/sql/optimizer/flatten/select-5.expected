SELECT test.child.name AS name FROM test.child WHERE (test.child.pid IN (SELECT test.parent.id AS id FROM test.parent, test.child WHERE (test.parent.name = 'foo') AND TRUE)) AND TRUE
